roles/georchestra/templates/datafeeder/datafeeder.properties.j2

```diff
@@ -41,7 +41,7 @@ datafeeder.publishing.geoserver.api-url=http://localhost:{{ tomcat_instances['ge
 datafeeder.publishing.geoserver.public-url=https://${domainName}/geoserver
 # Use this for HTTP basic authentication to geoserver api url:
 #datafeeder.publishing.geoserver.auth.type=basic
-#datafeeder.publishing.geoserver.auth.basic.username=geoserver_privileged_user
+#datafeeder.publishing.geoserver.auth.basic.username=georchestra_privileged_user
```

Comment:
this section might require a second look

Reply:
it is commented out, but switching to B-A instead of "sec"-header based authentication is unsafe for now:
https://github.com/georchestra/georchestra/issues/3930

Reply:
Yeah, thanks for the heads up.
I meant the whole file has to be in sync with the datadir upstream.
It currently diverges on `datafeeder.publishing.geoserver.auth.headers.[sec-username]` and `datafeeder.publishing.geonetwork.auth.headers.[sec-username]`

Reply:
I made the modification